movement/watch_faces/sensor/medical_thermometer_face.h

```diff
@@ -0,0 +1,71 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2024 Christian Buschau
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+#ifndef MEDICAL_THERMOMETER_FACE_H_
+#define MEDICAL_THERMOMETER_FACE_H_
+
+#include "movement.h"
+
+/*
+ * MEDICAL THERMOMETER
+ *
+ * This watch face shows the current temperature in degrees Celsius. Press and
+ * hold the alarm button to toggle between Celsius and Fahrenheit. Press and
+ * release the alarm button to start a "timer". The watch will sound an alarm
+ * when the temperature remains constant for at least 30 seconds and the
+ * temperature will stop updating until you press the alarm button. You can
+ * cancel the alarm by pressing the button again. If the temperature doesn't
+ * remain constant until the low energy timeout is reached, the alarm will stop.
```

Comment:
Just the alarm? Will the face continue the temperature fix process in low energy mode? I ask because I use a ten second low energy timeout. After reading the code, to me it looks like it continues to read out temperatures even in low energy mode. Please confirm if that's you intention. Others may have concerns about power usage.

Reply:
It continues to read the temperature in low energy mode and updates every five minutes. That's the same behavior as the temperature readout face. The default minimum timeout for low energy mode is ten minutes, for ten seconds, this won't work.